docs/getting_started.md

```diff
@@ -2,6 +2,9 @@
 
 This section provides information on next steps you might take after downloading a dataset from the ScPCA portal.
 
+We have also provided a workflow that can be used to perform initial filtering, dimensionality reduction, normalization, and clustering on single-cell and single-nuclei samples in parallel.
+This workflow and more resources for processing single-cell datasets can be found on our Github in the [`scpca-downstream-analyses` repository](https://github.com/AlexsLemonade/scpca-downstream-analyses).
```

Comment:
```suggestion
This workflow and more resources for processing single-cell/single-nuclei datasets can be found on our Github in the [`scpca-downstream-analyses` repository](https://github.com/AlexsLemonade/scpca-downstream-analyses).
```

Reply:
Or something like the above suggestion since you do mention both types of samples in the first sentence there?